common/changes/@microsoft.azure/openapi-validator-rulesets/rmanda-fix_rpc_patch_v1_02_2023-05-12-05-42.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
```

Comment:
maybe prefix this file with `rkmanda` not `rmanda` to avoid confusion on user names.

Reply:
the nameis automatically derived based on the branch name used. I will use my github handle next time in the branch name. will leave this as is for now